--- enscript-1.6.6/src/main.c.orig	2011-10-30 17:48:42.000000000 +0100
+++ enscript-1.6.6/src/main.c	2012-09-30 21:24:10.177497456 +0200
@@ -329,6 +329,8 @@
  * Send mail notification to user after print job has been completed.
  */
 int mail = 0;
+char *mail_name;
+
 
 /*
  * -M, --media
@@ -848,7 +850,7 @@
   {"no-page-prefeed",		no_argument,		0, 'K'},
   {"lineprinter",		no_argument,		0, 'l'},
   {"lines-per-page",		required_argument,	0, 'L'},
-  {"mail",			no_argument,		0, 'm'},
+  {"mail",			optional_argument,	0, 'm'},
   {"media",			required_argument,	0, 'M'},
   {"copies",			required_argument,	0, 'n'},
   {"newline",			required_argument,	0, 'N'},
@@ -1843,7 +1845,11 @@
       /* Format spooler options. */
       spooler_options[0] = '\0';
       if (mail)
-	strcat (spooler_options, "-m ");
+	{
+	  strcat (spooler_options, "-m ");
+	  strcat (spooler_options, mail_name);
+	  strcat (spooler_options, " ");
+	}
       if (no_job_header)
 	{
 	  strcat (spooler_options, no_job_header_switch);
@@ -2006,7 +2012,7 @@
       int i;
 
       c = getopt_long (argc, argv,
-		       "#:123456789a:A:b:BcC::d:D:e::E::f:F:gGhH::i:I:jJ:kKlL:mM:n:N:o:Op:P:qrRs:S:t:T:u::U:vVw:W:X:zZ",
+		       "#:123456789a:A:b:BcC::d:D:e::E::f:F:gGhH::i:I:jJ:kKlL:m::M:n:N:o:Op:P:qrRs:S:t:T:u::U:vVw:W:X:zZ",
 		       long_options, &option_index);
 
       if (c == -1)
@@ -2207,6 +2213,10 @@
 
 	case 'm':		/* send mail upon completion */
 	  mail = 1;
+	  if(optarg)
+	    mail_name = (optarg);
+	  else
+	    mail_name = (*passwd).pw_name;
 	  break;
 
 	case 'M':		/* select output media */
