--- enscript-1.6.4/lib/Makefile.am.orig	2003-03-05 08:26:29.000000000 +0100
+++ enscript-1.6.4/lib/Makefile.am	2003-05-09 07:52:40.000000000 +0200
@@ -38,12 +38,8 @@
 all-local: enscript.cfg
 
 install-data-local: enscript.cfg
-	$(top_srcdir)/mkinstalldirs $(sysconfdir)
-	if test -r $(sysconfdir)/enscript.cfg; then \
-	  cp $(sysconfdir)/enscript.cfg $(sysconfdir)/enscript.cfg.old; \
-	else :; \
-	fi
-	$(INSTALL_DATA) enscript.cfg $(sysconfdir)/enscript.cfg
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(sysconfdir)
+	$(INSTALL_DATA) enscript.cfg $(DESTDIR)$(sysconfdir)/enscript.cfg
 
 uninstall-local:
 	rm -f $(sysconfdir)/enscript.cfg
--- enscript-1.6.4/afm/Makefile.am.orig	2003-03-05 08:26:16.000000000 +0100
+++ enscript-1.6.4/afm/Makefile.am	2003-05-09 07:53:56.000000000 +0200
@@ -36,10 +36,10 @@
 EXTRA_DIST = font.map $(default_afm) $(public_fonts)
 
 install-data-local:
-	$(top_srcdir)/mkinstalldirs $(datadir)/enscript
-	$(top_srcdir)/mkinstalldirs $(datadir)/enscript/afm
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(datadir)/enscript
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(datadir)/enscript/afm
 	for f in $(EXTRA_DIST); do \
-	  $(INSTALL_DATA) $(srcdir)/$$f $(datadir)/enscript/afm/$$f; \
+	  $(INSTALL_DATA) $(srcdir)/$$f $(DESTDIR)$(datadir)/enscript/afm/$$f; \
 	done
 
 uninstall-local:
--- enscript-1.6.4/states/hl/Makefile.am.orig	2003-03-05 08:26:33.000000000 +0100
+++ enscript-1.6.4/states/hl/Makefile.am	2003-05-09 07:54:20.000000000 +0200
@@ -45,10 +45,10 @@
 EXTRA_DIST = $(states)
 
 install-data-local:
-	$(top_srcdir)/mkinstalldirs $(datadir)/enscript
-	$(top_srcdir)/mkinstalldirs $(datadir)/enscript/hl
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(datadir)/enscript
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(datadir)/enscript/hl
 	for f in $(states); do \
-	  $(INSTALL_DATA) $(srcdir)/$$f $(datadir)/enscript/hl/$$f; \
+	  $(INSTALL_DATA) $(srcdir)/$$f $(DESTDIR)$(datadir)/enscript/hl/$$f; \
 	done
 
 uninstall-local:
