--- enscript-1.6.2/configure.in.orig	Wed Mar 20 20:34:01 2002
+++ enscript-1.6.2/configure.in	Wed Mar 20 20:39:28 2002
@@ -25,16 +25,16 @@
 dnl Check some functions.
 AC_CHECK_FUNCS(strchr)
 AC_CHECK_FUNC(strerror, AC_DEFINE(HAVE_STRERROR), [
-  LIBOBJS="$LIBOBJS strerror.o"
+  AC_LIBOBJ([strerror])
 ])
 AC_CHECK_FUNC(memmove, AC_DEFINE(HAVE_MEMMOVE), [
-  LIBOBJS="$LIBOBJS memmove.o"
+  AC_LIBOBJ([memmove])
 ])
 AC_CHECK_FUNC(memcpy, AC_DEFINE(HAVE_MEMCPY), [
-  LIBOBJS="$LIBOBJS memcpy.o"
+  AC_LIBOBJ([memcpy])
 ])
 AC_CHECK_FUNC(strtoul, AC_DEFINE(HAVE_STRTOUL), [
-  LIBOBJS="$LIBOBJS strtoul.o"
+  AC_LIBOBJ([strtoul])
 ])
 
 AC_CHECK_FUNC(getcwd, AC_DEFINE(HAVE_GETCWD), [
